### PR TITLE
feature(coral-admin): Add users role change

### PR DESCRIPTION
**Pull request by impronunciable**
It adds a change role select for every user :)

### CODE REVIEW

File: client/coral-admin/src/translations.js [hunk 1]
```diff
@@ -30,7 +34,11 @@ export default {
   es: {
     'community': {
       username_and_email: 'Usuario y E-mail',
-      account_creation_date: 'Fecha de creación de la cuenta'
+      account_creation_date: 'Fecha de creación de la cuenta',
+      newsroom_role: 'Rol en la redacción',
+      admin: 'Administrador',
+      moderator: 'Moderador',
+      role: 'Select role...'
```
Comment (okbel) [inline client/coral-admin/src/translations.js:41]: `role` should be `Seleccionar rol` ?

File: client/coral-admin/src/containers/Community/Table.js [hunk 2]
```diff
@@ -1,34 +1,66 @@
...
-);
+const lang = new I18n(translations);
 
-export default Table;
+class Table extends Component {
+
+  constructor (props) {
+    super(props);
+    this.onRoleChange = this.onRoleChange.bind(this);
+  }
```
Comment (okbel) [inline client/coral-admin/src/containers/Community/Table.js:16]: All the functionality of Community is in the container `/containers/Community/CommunityContainer.js`. Can we keep this one stateless? It's not mandatory. So we avoid connecting and classing this one too. (I don't mind doing it)

**Comment by okbel** [comment]
The Select is not changing when I select an Option. (`SET_ROLE` it's being triggered)

out

**Review by okbel — COMMENTED** [review]
Added a few comments. Let me know what you think :)

**Review by okbel — APPROVED** [review]

**Comment by riley** [comment]
excellent, works for me 💯

**Pull request merged**